week-1-practise/level-3/1-piping.js

```diff
@@ -11,21 +11,21 @@
 */
 
 var startingValue = 5;
-
+var result = format(startingValue * 2 + 10);
```

Comment:
It get's the right answer, but not in quite the way the question is looking for. You have copied what the `add` and `multiply` functions do, but the question would like your to _call_ the functions directly, like how you have called the `format` function.